src/graph.ts

```diff
@@ -402,7 +405,6 @@ export class PixiGraph<NodeAttributes extends BaseNodeAttributes = BaseNodeAttri
     this.viewport.pause = true; // disable viewport dragging
 
     document.addEventListener('mousemove', this.onDocumentMouseMoveBound);
-    document.addEventListener('mouseup', this.onDocumentMouseUpBound, { once: true });
```

Comment:
When dragging of nodes is disabled, if you hold the mouse on the specific node and move it somewhere else (which will move the canvas, not the node), browser won't trigger a `mouseup` event on the node you initially pressed.

So that's why I moved the `mouseup` handler from `enableNodeDragging` function to the event handler of node's `mousedown` event.

This way, no matter where user's mouse has moved, we can always catch the `mouseup` event and unfocus the node. Otherwise node will stay in selected/focused state forever after hold & drag.